src/pages/Map.css

```diff
@@ -0,0 +1,70 @@
+.PinCard {
+  width: 250px;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.PinCardLabel {
+  width: max-content;
+  color: pink;
+  font-size: 15px;
+  border-bottom: 1px solid pink;
+  margin: 3px 0;
+}
+
+.PinCardDesc {
+  display: flex;
+  align-items: right;
+  font-size: 14px;
+}
+
+.PinStar {
+  display: flex;
+  align-items: right;
```

Comment:
`align-items: right` doesn't exist. You can remove this and it would still work the same way

Reply:
Valid values for this property in a flex:
https://css-tricks.com/almanac/properties/a/align-items/